@xen-orchestra/lite/src/views/new-vm/NewVmView.vue

```diff
@@ -545,7 +567,58 @@ const bootFirmwares = computed(() => [...new Set(templates.value.map(template =>
 
 const defaultSr = computed(() => pool.value!.default_SR)
 
-const filteredSrs = computed(() => srs.value.filter(sr => sr.content_type !== 'iso' && sr.physical_size > 0))
+const filteredSrs = computed(() => {
+  return srs.value.filter(sr => {
+    const pbdRef = getPbdByOpaqueRef(sr.PBDs[0])
```

Comment:
Here the `pbdRef` is the object, so `pbd` would be more appropriate:
```suggestion
    const pbd = getPbdByOpaqueRef(sr.PBDs[0])
```